client/src/components/Reviews.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+class Reviews extends React.Component {
+  static like(event) {
+    const reviewId = {
+      reviewId: event.target.parentNode.parentNode.getAttribute('reviewid'),
+      likes_count: event.target.previousSibling.textContent,
+    };
+
+    fetch('http://localhost:3003/review', {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(reviewId),
+    })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviews: [],
+    };
+    this.reviewRef = React.createRef();
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3003/reviews/20')
+      .then((result) => {
+        return result.json();
+      })
+      .then((result) => {
+        this.setState(() => (
+          {
+            reviews: result,
+          }
+        ));
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+
+  render() {
+    const { reviews } = this.state;
+
+    const renderedReviews = reviews.map(review => (
+      <div key={review._id} reviewid={review._id} ref={this.reviewRef} className="review">
+        <div className="review-header">
```

Comment:
Not sure if this is included in the code, but I wanted to note the "five-star" image that appears with each review. The number of stars "filled-in" in the image shows the rating of that user gave the book.

Reply:
Yeah, I decided making this into its own PR